Wikipedia/Categories/NSAttributedString+WMFModifyParagraphs.h

```diff
@@ -0,0 +1,20 @@
+//  Created by Monte Hurd on 8/17/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import <Foundation/Foundation.h>
+
+@interface NSAttributedString (WMFModifyParagraphs)
+
+/**
+ *  Sometimes you need to adjust all paragraph styles without destroying existing paragraph style properties.
```

Comment:
@montehurd if you install the VVDocumenter Plugin via alcatra, typing "///" will create the code block for you with all the boiler plate filled out.

https://github.com/onevcat/vvdocumenter-xcode